lib/api/ban-id/models.js

```diff
@@ -1,15 +1,8 @@
-import mongo from '../../util/mongo.cjs'
-
-const COLLECTION_ADDRESS = 'address_test'
-const COLLECTION_COMMON_TOPONYM = 'common_toponym_test'
-const COLLECTION_DISTRICT = 'district_test'
+import {Address, CommonToponym, District} from '../../util/sequelize.js'
 
 export async function idsInDataBase(ids) {
-  return (
-    await Promise.all([
-      mongo.db.collection(COLLECTION_ADDRESS).find({id: {$in: ids}}).toArray(),
-      mongo.db.collection(COLLECTION_COMMON_TOPONYM).find({id: {$in: ids}}).toArray(),
-      mongo.db.collection(COLLECTION_DISTRICT).find({id: {$in: ids}}).toArray()
-    ])
-  ).flat()
+  const addressResults = await Address.findAll({where: {id: ids}, raw: true})
+  const commonToponymResults = await CommonToponym.findAll({where: {id: ids}, raw: true})
+  const districtResults = await District.findAll({where: {id: ids}, raw: true})
+  return [...addressResults, ...commonToponymResults, ...districtResults]
```

Comment:
```suggestion
  return (
    await Promise.all([
        Address.findAll({where: {id: ids}, raw: true}),
        CommonToponym.findAll({where: {id: ids}, raw: true}),
        District.findAll({where: {id: ids}, raw: true}),
      ])
    ).flat()
  )
```

L'usage du Promise.all permet de lancer les 3 requêtes en même temps, plutôt que l'une après l'autre. Cela permet donc de gagner du temps de réponse.